tensorflow/training/docker/2.2.0/py3/Dockerfile.cpu

```diff
@@ -121,11 +121,12 @@ RUN wget -c https://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz \
  && cd openssl-${OPENSSL_VERSION} \
  && ./config && make && make test \
  && make install \
+ && ldconfig \
```

Comment:
this is also done in line 62

Reply:
we need to refresh the cache again using ldconfig once openssl is installed so we are using it for the second tests